wasm/jit/jit_arm64_test.go

```diff
@@ -938,3 +961,322 @@ func TestArm64Compiler_compieleDrop(t *testing.T) {
+				requirePushTwoInt32Consts(t, x1, x2, compiler)
+				// Le on unsigned integer produces the value on COND_LS register.
+				err := compiler.compileLe(&wazeroir.OperationLe{Type: wazeroir.SignedTypeUint32})
+				require.NoError(t, err)
+			},
+		},
+		{
+			name: "LE",
+			setupFunc: func(t *testing.T, compiler *arm64Compiler, shoulGoElse bool) {
+				x1, x2 := uint32(1), uint32(2)
+				if shoulGoElse {
+					x2, x1 = x1, x2
+				}
+				requirePushTwoInt32Consts(t, x1, x2, compiler)
+				// Le on signed integer produces the value on COND_LE register.
+				err := compiler.compileLe(&wazeroir.OperationLe{Type: wazeroir.SignedTypeInt32})
+				require.NoError(t, err)
+			},
+		},
+		// {name: "EQ"} TODO: after compileEq support
+		// {name: "NE"} TODO: after compileNe support
+		{
+			name: "HS",
+			setupFunc: func(t *testing.T, compiler *arm64Compiler, shoulGoElse bool) {
+				x1, x2 := uint32(2), uint32(1)
+				if shoulGoElse {
+					x2, x1 = x1, x2
+				}
+				requirePushTwoInt32Consts(t, x1, x2, compiler)
+				// Ge on unsigned integer produces the value on COND_HS register.
+				err := compiler.compileGe(&wazeroir.OperationGe{Type: wazeroir.SignedTypeUint32})
+				require.NoError(t, err)
+			},
+		},
+		{
+			name: "GE",
+			setupFunc: func(t *testing.T, compiler *arm64Compiler, shoulGoElse bool) {
+				x1, x2 := uint32(2), uint32(1)
+				if shoulGoElse {
+					x2, x1 = x1, x2
+				}
+				requirePushTwoInt32Consts(t, x1, x2, compiler)
+				// Ge on signed integer produces the value on COND_GE register.
+				err := compiler.compileGe(&wazeroir.OperationGe{Type: wazeroir.SignedTypeInt32})
+				require.NoError(t, err)
+			},
+		},
+		{
+			name: "HI",
+			setupFunc: func(t *testing.T, compiler *arm64Compiler, shoulGoElse bool) {
+				x1, x2 := uint32(2), uint32(1)
+				if shoulGoElse {
+					x2, x1 = x1, x2
+				}
+				requirePushTwoInt32Consts(t, x1, x2, compiler)
+				// Gt on unsigned integer produces the value on COND_HI register.
+				err := compiler.compileGt(&wazeroir.OperationGt{Type: wazeroir.SignedTypeUint32})
+				require.NoError(t, err)
+			},
+		},
+		{
+			name: "GT",
+			setupFunc: func(t *testing.T, compiler *arm64Compiler, shoulGoElse bool) {
+				x1, x2 := uint32(2), uint32(1)
+				if shoulGoElse {
+					x2, x1 = x1, x2
+				}
+				requirePushTwoInt32Consts(t, x1, x2, compiler)
+				// Gt on signed integer produces the value on COND_GT register.
+				err := compiler.compileGt(&wazeroir.OperationGt{Type: wazeroir.SignedTypeInt32})
+				require.NoError(t, err)
+			},
+		},
+		{
+			name: "LO",
+			setupFunc: func(t *testing.T, compiler *arm64Compiler, shoulGoElse bool) {
+				x1, x2 := uint32(1), uint32(2)
+				if shoulGoElse {
+					x2, x1 = x1, x2
+				}
+				requirePushTwoInt32Consts(t, x1, x2, compiler)
+				// Lt on unsigned integer produces the value on COND_LO register.
+				err := compiler.compileLt(&wazeroir.OperationLt{Type: wazeroir.SignedTypeUint32})
+				require.NoError(t, err)
+			},
+		},
+		{
+			name: "LT",
```

Comment:
this makes me feel we can do compileLT instead of compileLt

Reply:
so I will keep as-is but renaming to compilerLT works for me in anyway